lib/internal/modules/cjs/context.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const { Buffer } = require('buffer');
```

Comment:
It does not seem to worth a lot to create a separate module for this - this could be added as a special case in `NativeModule.require` or just a special property that hangs off `NativeModule`

Reply:
Sure, I've added this.